src/DependencyInjection/MercureExtension.php

```diff
@@ -67,9 +74,35 @@ public function load(array $configs, ContainerBuilder $container)
             $bus = $hub['bus'] ?? null;
             $attributes = null === $bus ? [] : ['bus' => $hub['bus']];
             $publisherDefinition->addTag('messenger.message_handler', $attributes);
+
+            if ($enableProfiler) {
+                $container->register("$hubId.traceable", TraceablePublisher::class)
+                    ->setDecoratedService($hubId)
+                    ->addArgument(new Reference("$hubId.traceable.inner"))
+                    ->addArgument(new Reference('debug.stopwatch'));
+
+                $publishers[$name] = new Reference("$hubId.traceable");
+                if ($defaultHub === $hubId) {
+                    $container->setAlias(TraceablePublisher::class, "$hubId.traceable");
```

Comment:
do we need this alias? I mean, nobody should typehint against `TraceablePublisher` and the `PublisherInterface` type should naturally be wired with the traceable default hub thanks to decoration, or am I mistaken?